lib/gceimgutils/gceutils.py

```diff
@@ -187,3 +189,53 @@ def _no_name_warning(image, log_callback):
     msg = 'WARNING: Found image with no name, ignoring for search results. '
     msg += 'Image ID: %s' % image['ImageId']
     log_callback.info(msg)
+
+
+# ----------------------------------------------------------------------------
+def image_to_dict(image):
+    return {
```

Comment:
I left the keys camel case to match the existing API output. However, if we push a major release then we could use snake case and thus the dictionary could then be used to create an object of Image class.